### PR TITLE
Allow upgrading to a newer GV major version on A-C main

### PR DESCRIPTION
**Pull request by jcristau**
To streamline merge days we want relbot to bump GV to a new major
version when it becomes available instead of having to wait and do it
manually.

### CODE REVIEW

**Comment by jcristau** [comment]
cc @rvandermeulen FYI

File: src/android_components.py [hunk 1]
```diff
@@ -150,7 +150,10 @@ def _update_geckoview(
             f"{ts()} Current GV {gv_channel.capitalize()} version in A-C {ac_repo.full_name}:{release_branch_name} is {current_gv_version}"
         )
 
-        current_gv_major_version = major_gv_version_from_version(current_gv_version)
+        if ac_major_version == "main":
+            current_gv_major_version = None
```
Comment (JohanLorenzo) [inline src/android_components.py:154]: Nit: Let add a small comment to explain why we're giving this special value. For instance:

```suggestion
            # We always want to be on the latest geckoview version on the main branch.
            current_gv_major_version = None
```

**Review by JohanLorenzo — APPROVED** [review]
LGTM modulo a small nit!

File: src/android_components.py (hunk 1 above)
Comment (jcristau) [reply src/android_components.py:155]: Fair.  Added.

**Pull request merged**